Finally start updating the public roadmap (#2831)
DFE-Digital/register-early-career-teachers-public: documentation/site/content/product/roadmap/later.md

We will:

> * look at how we can reduce the complexity of schedules, so that we can simplify validation and make management of training easier for lead providers

Reviewer: suggest revising in case this is read by actors not familiar with the term schedules and also keeping it agnostic of the current solution.

Maybe: * look at how we can reduce the complexity of managing participants who don't follow the "standard" pattern of training, so that we can simplify validation and make management of training easier for lead providers

Author: yep, agree this is simpler english

> * improve how withdrawals and deferrals of ECTS and mentors are reported by schools and lead providers, to reduce workload and duplication for both users

Reviewer 2: ```suggestion
* improve how withdrawals and deferrals of ECTs and mentors are reported by schools and lead providers, to reduce workload and duplication for both users
```

> * improving how statuses work over the API, to make it clearer for lead providers the current state of a participant

Reviewer 2: ```suggestion
* improve how statuses work over the API, to make it clearer for lead providers the current state of a participant
```

We will:

> * look at how we can reduce the complexity of schedules, so that we can simplify validation and make management of training easier for lead providers

Author: ```suggestion
* look at how we can reduce the complexity of managing participants who don't follow the "standard" pattern of training, so that we can simplify validation and make management of training easier for lead providers
```
